geojson2csv/samples.geojson

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I guess this file is only for local testing?

Reply:
Yes, removing this for now. Easy to add when want to add testing.